src/containers/Search/index.tsx

```diff
@@ -13,6 +13,7 @@ import styles from './styles'
 import SearchIcon from '@mui/icons-material/Search'
 import { parseNode, parseNodes } from './graph.utils'
 import { TextField } from '@mui/material'
+import FeedClaim from '../../feedCLAIM'
```

Comment:
please do not use upper case in file names, its better to have all filenames lower case please

Reply:
no, a file called index , FeedClaim is a functional component @gvelez17